packages/lesswrong/components/posts/PostsPage/ContentType.tsx

```diff
@@ -29,6 +29,10 @@ const styles = (theme: ThemeType): JssStyles => ({
   tooltipTitle: {
     marginBottom: 8,
   },
+  sectionTitle: {
+    fontSize: 14,
+    marginBottom: -4,
```

Comment:
Can we try to avoid negative margins? I think you can accomplish the same thing by making `marginBottom: 4` and setting the `noBottomPadding` option on `<SectionTitle>`?

Reply:
Makes sense here so I've gone ahead and made the change. I don't really see the argument against negative margins in the general case though?